### ISSUE TITLE
Reported download speed fluctuates rapidly

### ISSUE THREAD

**Issue by NotMithical** [jc3213/download_with_aria2#87]
I'm honestly not exactly sure how to describe what's happening here. It looks like the speed is constantly climbing causing it to overflow and go back down to -2,147,483,647 and then continues climbing.

https://github.com/jc3213/download_with_aria2/assets/95518825/c6af5304-4fdc-47e9-adcf-e4531ae94e22

**Comment by jc3213** [comment]
What's your options? I can't produce your problem using default options

**Comment by NotMithical** [comment]
I actually haven't been able to produce this issue again either, it must have been something on my end, possibly with aria2 itself. I apologize, I should've tested more extensively before opening an issue.

**Comment by jc3213** [comment]
The weird behavior in your video is just like that you have set the `manager_interval` to a very low value, or maybe you've updated from an very olde version.

I've changed `manager_interval` in [4.8.1](https://github.com/jc3213/download_with_aria2/releases/tag/4.8.1.2493),  the value is changed from 10000 to 10. I've made a small patch for this change, but it will take some time to take effect.

It maybe the patch haven't work correctly when you updated the extension, But when you restart the extension/Browser. The new logic is working and the problem has been "fixed". That maybe my problem, sorry.
